### PR TITLE
Use .env file for environment variables

### PR DESCRIPTION
**Pull request by ajmaddaford**
### What is the context of this PR?

Adds a .development.env file with developer defaults. This provides one
common place to set defaults (used both locally and in Docker) and
should stop people having environment issues when new variables are
added to runner. It should also mean less configuration for code editors.

Also introduces a Makefile for running the various commands - less typing
and tab completion on compatible shells.

The `jwt-test-secrets` have been removed as we default to using the built `dev-secrets.yml`. The test keys used by the tests have been moved to `tests/data/jwt-test-keys`.

### How to review 
- Try following the updated Readme to run locally and in Docker
- Try all the commands in the Makefile
- Try integrating with your code editor of choice

### CODE REVIEW

File: .development.env [hunk 1]
```diff
@@ -0,0 +1,18 @@
+FLASK_APP=application.py
+FLASK_ENV=development
+EQ_DEVELOPER_LOGGING=True
+EQ_SUBMITTED_RESPONSES_TABLE_NAME=dev-submitted-responses
+EQ_QUESTIONNAIRE_STATE_TABLE_NAME=dev-questionnaire-state
+EQ_SESSION_TABLE_NAME=dev-eq-session
+EQ_USED_JTI_CLAIM_TABLE_NAME=dev-used-jti-claim
+EQ_ENABLE_SECURE_SESSION_COOKIE="False
+EQ_ENABLE_LIVE_RELOAD=True
+EQ_RABBITMQ_ENABLED=False
```
Comment (jonnyshaw89) [inline .development.env]: this is an old var. I cant find it used anywhere

**Review by jonnyshaw89 — COMMENTED** [review]
👍 Lots of deletes

**Comment by MebinAbraham** [comment]
Makefile finally 👍

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+test-functional:
+	pipenv run ./scripts/run_tests_functional.sh
+
+run:
+	ln -sf .development.env .env
+	pipenv run flask run
+
+dev-compose-up:
+	docker-compose -f docker-compose-dev.yml up -d
```
Comment (jonnyshaw89) [inline Makefile:21]: You need to change references to `go-launch-a-survey` to `eq-questionnaire-launcher` in both compose files.
@bitdivision must have missed them when forking the launcher repo

**Review by jonnyshaw89 — CHANGES_REQUESTED** [review]
Also as spoken about IRL the keys are missing for the integration tests to launch

**Comment by jonnyshaw89** [comment]
`EQ_PROFILING` is left behind in `settings.py` and `README.md`

File: README.md [hunk 3]
```diff
@@ -428,7 +407,9 @@ There is a python script for generating tokens for use in development, to run:
 python token_generator.py
 ```
 
-### Profiling
+---
+
+## Profiling
```
Comment (jonnyshaw89) [inline README.md:412]: This section needs updating

**Review by jonnyshaw89 — APPROVED** [review]

**Review by MebinAbraham — APPROVED** [review]
👍

**Pull request merged**